packages/data-stores/src/plans/constants.ts

```diff
@@ -17,18 +22,45 @@ interface Currency {
 	decimal: number;
 }
 
-export const plansOrder = [ PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE ];
+export const plansOrder = [
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_ECOMMERCE,
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
+];
+
+export const billedYearlySlugs = [
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_ECOMMERCE,
+] as const;
+
+export const billedMonthlySlugs = [
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
+] as const;
 
-export type PlanPath = 'free' | 'personal' | 'premium' | 'business' | 'ecommerce';
+export const plansPaths = [ 'free', 'personal', 'premium', 'business', 'ecommerce' ] as const;
 
-export const plansPaths: PlanPath[] = [ 'free', 'personal', 'premium', 'business', 'ecommerce' ];
+export type PlanPath = typeof plansPaths[ number ];
 
 export const plansProductSlugs = [
 	PLAN_FREE,
 	PLAN_PERSONAL,
 	PLAN_PREMIUM,
 	PLAN_BUSINESS,
 	PLAN_ECOMMERCE,
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
 ];
```

Comment:
Just a suggestion to reduce repetition and make things a bit easier to grasp

```suggestion
export const plansProductSlugs = [
	PLAN_FREE,
	... plansOrder,
];
```